srcs/server/Server.cpp

```diff
@@ -164,19 +183,26 @@ void	Server::handleNewConnection(void) {
 }
 
 void	Server::dropConnection(int i) {
-	// Remove from the pfds list by copying the file descriptor of the last one over it. 
-    _connections[_pfds[i].fd]->closeSocket();
-    _pfds[i] = _pfds[_fd_count - 1];
-	_fd_count--;
 
```

Comment:
line 186 - in the `catch` statement - you are missing a reference (`&`) to the `std::exception e`